### DISCUSSION TITLE
where is apohl79 is this project dead. Lets keep this thing going

### DISCUSSION THREAD

**Discussion by poppagreg** [apohl79/audiogridder#1404] [General]
Is there ever going to be an update or fixes for this project. Lets keep this going where is apohl? what is happening. Any developers want to get on this?

**Comment by AnietyDev** [comment]
A better tone than this post might have been:

'Hey Apohl79, just checking in. Would love to know your plans (if any) for AudioGridder going forward. I love the tool you created! I'm specifically wondering about XYZ.'

AudioGridder is working fine for me and most of the issue posts I see are complex plugins that probably don't lend themselves to working well with AudioGridder. I know I wouldn't bother to address them in a paid product let alone donation ware. 

It looks like Apohl79 did a commit in September so he's still around.

**Reply by apohl79** [reply]
Hey, I know it's been a while since I could focus on AG. The project is not dead, I'm just fully occupied with my job right now... but it will go on here.

**Reply by poppagreg** [reply]
Great to hear, was starting to wonder. It's a great idea and there is so much potential with it. Yes please keep this going. Thanks

**Reply by poppagreg** [reply]
> A better tone than this post might have been:
> 
> 'Hey Apohl79, just checking in. Would love to know your plans (if any) for AudioGridder going forward. I love the tool you created! I'm specifically wondering about XYZ.'
> 
> AudioGridder is working fine for me and most of the issue posts I see are complex plugins that probably don't lend themselves to working well with AudioGridder. I know I wouldn't bother to address them in a paid product let alone donation ware.
> 
> It looks like Apohl79 did a commit in September so he's still around.

You know what they say about opinions and you know where you can stick yours.

**Reply by Loupap** [reply]
Hey @apohl79 

Just wanted to reach out and say thanks for Audiogridder. My entire studio setup is totally dependent on it and I love it.

I am running a total of four servers in addition to my main DAW computer and I absolutely love the flexibility it gives me with my clients.

So glad to hear that Audiogridder is not dead!!

Thanks,
Lou

**Reply by poppagreg** [reply]
wow 4 servers that is crazy, why so many?

**Reply by kcoul** [reply]
I had been planning a new feature set for sometime now but same thing happened, got overwhelmed by my day job. Now I have even more reason to implement the feature set to use in an important new project so it's finally going to happen. 

So I will be seeing through the new feature set (to be able to use the audio and MIDI I/O of the server itself) in the next few months!
If there are any other serious outstanding issues, I will try and get them resolved at the same time for a new release in 2025.

**Reply by ralphonz** [reply]
This is good to read, I'm just discovering AudioRigger now and it does seem a little bit quiet around here!  I'm going to try it out today and if it works well, I can't believe there isn't more of a buzz or hype around this.

**Reply by poppagreg** [reply]
Well 2025 is coming to an end hopefully we will see something soon on this project.
